src/vs/workbench/browser/layout.ts

```diff
@@ -18,7 +18,10 @@ import {IContextViewService} from 'vs/platform/contextview/browser/contextView';
 import {IEventService} from 'vs/platform/event/common/event';
 
 const DEFAULT_MIN_PART_WIDTH = 170;
+const DEFAULT_MIN_PANEL_PART_HEIGHT = 170;
 const HIDE_SIDEBAR_WIDTH_THRESHOLD = 50;
+const HIDE_PANEL_HEIGHT_THRESHOLD = 50;
```

Comment:
@bpasero no. I wanted to have the nice action you have in sidebar to hide it when this threshold is reached but not sure if we want it since bringing it back would not be as easy as in the sidebar. Or you think that whole behavior makes sense for panel as well?